include/event_manager.h

```diff
@@ -283,6 +283,23 @@ void event_manager_trace_event_submission(const struct event_header *eh,
  **/
 int event_manager_trace_event_init(void);
 
+/** Allocate event.
```

Comment:
Do we want to expose these functions in the Event Manager API ?

Reply:
We can move them to event_manager_priv.h, but I thought that if user has a possibility to overwrite them they should be in public Api, but I can have it wrong.

Reply:
I think this is similar case as tracing function. I would leave it here. The only thing that could be improved is to document arguments and return.